test/oas2tson.test.ts

```diff
@@ -257,4 +257,270 @@ tap.test('oas2tson', async t => {
       'Customer.ts is created correctly'
     )
   })
+
+  t.test('runCommand function with --path', async t => {
```

Comment:
With all you changed, I'm not sure that this unique test will be enough to cover it

Reply:
I took that from the previous attempt to get `-p paths` working, which was reverted because it assumed hard camel case naming.

From a lines coverage perspective, `tap` is reporting slightly better coverage than before because I removed `tsType` in `adaptSchema` (not needed; may fail to compile in `ajv` strict mode). 

Are you looking for more functional coverage? If so, which `-p` keywords would you like tested?

I think most keywords in the `components` hierarchy are covered by `components.schemas`, but I can add a couple of others if you think they're distinct enough.

If it helps, the changes in `oas2tson.ts` come down to:

- Generating valid JS/TS names. (Many small helper functions, regexps, etc.)
- `isForOutput` to control items added to `outputSchemasMetaData`

Much of `oas2tson` is about naming, though. So, yes, many things changed to get names right so dereferencing would work. :smile:

Maybe the `pet` path would be a better option than `store/order`? The `pet` path includes `$ref`s to schemas, an example, a request body, and a response. They'll be dereferenced in `paths__pet` but must generate as expected if `pet` matches.